website/assets/style/code.scss

```
@@ -64,3 +64,9 @@ code {
         background-image: url(./res/copy-error.svg);
     }
 }
+
+@media (max-width: 500px) {
```

Lets use our predefined breakpoints. 

```suggestion
@include media-breakpoint-down(md){
```